fetchcode/package.py

```diff
@@ -91,7 +91,8 @@ def get_cargo_data_from_purl(purl):
     )
     versions = response.get("versions", [])
     for version in versions:
-        version_purl = PackageURL(type=purl.type, name=name, version=version.get("num"))
+        version_purl = PackageURL(
```

Comment:
Please don't change these lines :), they are out of scope of this PR

Reply:
Actually I have a setting of `Format on save` on my vscode, it happened unknowingly, will take care in the future. Thanks for the feedback.